source/index.rst

```diff
@@ -20,8 +20,11 @@ are roughly designed as single, guided courses.  Hands-on SciComp
 links to all of these materials and organizes them for self-study and
 determining where you need to start and where you need to end up.
 
+This course is coordinated by `Aalto University Science-IT`_ 
```

Comment:
Should we also add a direct link to contact info, or the scip email address?

Reply:
acknowledged